tests/events/ext.keboola.stream.sourceEnable.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": 5572396066,
+    "event": "ext.keboola.stream.sourceEnable",
+    "component": "keboola.stream.sourceEnable",
+    "message": "Source undeleted.",
+    "description": "",
+    "type": "info",
+    "runId": null,
+    "created": "2024-10-01T17:15:20+0200",
+    "configurationId": null,
+    "objectId": "",
+    "objectName": "",
+    "objectType": "",
+    "context": {
+        "remoteAddr": "20.253.172.73",
+        "httpReferer": null,
+        "httpUserAgent": "storage-reader",
+        "apiVersion": "v2"
+    },
+    "results": {
```

Comment:
why is it results and not params? 

Compare to https://github.com/keboola/event-schema/blob/main/tests/events/ext.keboola.vault.variableCreated.json

Reply:
I don't know... I just copied the older stream events from @Matovidlo.

Reply:
...who probably copied the structure from the old buffer event: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-buffer..json

Reply:
🤷 but it doesn't make sense. "I don't care if it doesn't make sense" is a valid answer, though. It's up to you. And maybe you should also ask someone from telemetry team if they care.

Reply:
I do care, I was just answering the question why it is that way...

However we're using Results here... https://github.com/keboola/keboola-as-code/blob/992fef0c1c99d4677c4c1ae945653f176e7d6f65/internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go#L164

So I'm guessing that to change this safely we need to duplicate the existing events here in the schema, have telemetry team use both for a while, then update our repository to use the new events, then let telemetry team know they can stop using the old events and only then finally remove the old events from the schema? It's a bit convoluted but otherwise we're risking issues, aren't we?

Reply:
Actually maybe not duplicate the events, just temporarily put the parameters into both params and results... but otherwise the workflow needs to be like that, right?

Reply:
Also note, that connection only validates events that it has a schema for.

Reply:
Alright, I'll get the service adjusted first to send both params and results then. :+1: Thanks for the clarification!